### PR TITLE
Feature/admin events participants

### PR DESCRIPTION
**Pull request by JuriMino**
In the admin page, which means events/index.blade, I created the link for participants.blade.php.
If the admin clicks the link, we can move to the page that all participants show.

### CODE REVIEW

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -16,14 +17,16 @@ class AdminController extends Controller
...
 
     const LOCAL_STORAGE_FOLDER = '/public/images/';
 
-    public function __construct(User $user, Event $event, Level $level)
+    public function __construct(User $user, Event $event, Level $level, Participant $participant)
     {
         $this->user = $user;
         $this->event = $event;
         $this->level = $level;
+        $this->participant = $participant;
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/AdminController.php:29]: is this being used?

File: app/Http/Controllers/AdminController.php [hunk 2]
```diff
@@ -15,15 +16,14 @@ class AdminController extends Controller
...
-    private $level;
+   
 
     const LOCAL_STORAGE_FOLDER = '/public/images/';
 
-    public function __construct(User $user, Event $event, Level $level)
+    public function __construct(User $user, Event $event)
     {
         $this->user = $user;
         $this->event = $event;
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/AdminController.php:26]: Is this being used also?

File: app/Http/Controllers/AdminController.php [hunk 3]
```diff
@@ -15,15 +16,14 @@ class AdminController extends Controller
...
     private $event;
-    private $level;
+   
 
     const LOCAL_STORAGE_FOLDER = '/public/images/';
 
-    public function __construct(User $user, Event $event, Level $level)
+    public function __construct(User $user, Event $event)
     {
         $this->user = $user;
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/AdminController.php:25]: also this one

Comment (JuriMino) [reply app/Http/Controllers/AdminController.php:25]: I'll fix it.

**Pull request merged**